torch/_inductor/fx_passes/joint_graph.py

```diff
@@ -21,20 +21,30 @@
 from .replace_random import replace_random_passes
 
 log = logging.getLogger(__name__)
-patterns = PatternMatcherPass()
+# We need two sets of PatternMatcherPasses here
+# One of them is for oneDNN Graph
+patterns = [PatternMatcherPass(), PatternMatcherPass()]
 
 
 @init_once_fakemode
-def lazy_init():
-    from .fuse_attention import _sfdp_init
+def lazily_init_onednn_graph():
```

Comment:
Why are we putting non onednn graph init under a function name with "onednn_graph"? Can we do something like
```python
def lazy_init():
    ...
    _onednn_graph_patterns_init()
    _pad_mm_init()
    _sfdp_init()
    _misc_patterns_init()
    ...
```

Reply:
Because we want to run the first `PatternMatcherPass` when the existing SDPA replacements have not been defined.

One pattern can't have multiple replacements in one pass & hits an assert.

Reply:
We can update `PatternMatcherPass` to handle having multiple replacements in the same pass. But we'd no longer need to do that with Jason's suggestions right ?

Reply:
Thanks for reviewing, @eellison!

Yes, it seems we could do that later, if necessary.
I'll submit a PR for accelerating MLPs using some of the code in this PR. It wouldn't be a PyTorch v2.3 feature, but will request your review on Monday.

This PR is blocked because if the ATen SDPA op is to be used, then the compute of larger attention patterns can't be offloaded to oneDNN Graph, which currently has hardcoded MHA patterns. 

Have a nice weekend!